### DISCUSSION TITLE
Learning Goals

### DISCUSSION THREAD

**Discussion by misik-eng** [MIT-Emerging-Talent/ET6-foundations-group-05#2] [General]
Learning Goals Discussion

**Comment by misik-eng** [comment]
Guys we can write down Our Personal Goals in here and the we can add it in our repository mark down:
 ->->collaboration/learning_goals.md
 
 I'll start first.
 1) My first personal goal in this project is gain collaboration skills and learn how a project works in real life.
 2)My another goal was meet and work with valuable people and my team are very active and every single one of them are awesome people. So .. Done

**Comment by aoyeoku** [comment]
Abbiihjah

1.) My first goal would be to enhance my time management skills and also become a lot more disciplined about my priorities.
2. My second would be to be very communicative with my team, whether it is when I have an issue, or I would like to help them out. Just being able to speak freely with my team without feeling anxious.

**Comment by doctorbanu** [comment]
Banu

1. My first goal is to gain experience in planning, organizing, and managing a project from start to finish. I will learn to create timelines, assign tasks, track progress.
2. My second goal is to improve my coding skills by collaborating with my team, learning best practices, and writing clean, efficient code.

**Comment by GeehanAli** [comment]
1\ My first goal is to master advanced data science techniques to contribute to impactful projects that drive social change and create real-world solutions.

2\ My second goal is to balance my pursuit of professional growth in data science with being a present and supportive parent to my twin boys by mastering time management and project planning skills, fostering their curiosity and resilience while advancing my own knowledge and expertise.

**Comment by imwaymaran** [comment]
# Learning Goals  

Before diving into our individual and collective goals, it's important to acknowledge the shared purpose driving this project. We aim to grow both as professionals and as team players by working together to solve challenges, learning from one another, and embracing collaboration as a cornerstone of success.  

## Collective Goals  
1. **Work Better as a Team**: Build strong teamwork by sharing knowledge, communicating openly, and supporting each other to hit our project goals.  
2. **Deliver Great Results**: Combine everyone’s skills to complete the project on time, meeting all the requirements and keeping the quality high.  
3. **Learn Together**: Help each other grow by improving technical skills, learning new ways to manage tasks, and becoming better at working as a team.  

---

## Individual Goals  

### Abbiihjah Oyeoku  
1. Enhance my time management skills and become more disciplined about my priorities.  
2. Be more communicative with my team, whether it’s to address an issue or offer help, and feel comfortable speaking freely without anxiety.  

### Banu Özyılmaz  
1. Gain experience in planning, organizing, and managing a project from start to finish, including creating timelines, assigning tasks, and tracking progress.  
2. Improve my coding skills by collaborating with my team, learning best practices, and writing clean, efficient code.  

### Geehan Ali  
1. Master advanced data science techniques to contribute to impactful projects that drive social change and create real-world solutions.  
2. Balance my pursuit of professional growth in data science with being a present and supportive parent to my twin boys by mastering time management and project planning skills, fostering their curiosity and resilience while advancing my own knowledge and expertise.  

### Gennadii Ershov  
1. **Improve Collaboration Skills**: Get better at working with different teams by communicating clearly, handling conflicts calmly, and contributing to shared goals.  
2. **Structure Knowledge**: Organize what I’ve learned in IT so far and start applying it through projects, certifications, and solving real-world problems.  

### Muhammet Isik  
1. Gain collaboration skills and learn how projects work in real life.  
2. Meet and work with valuable, active, and inspiring team members. (Done!)
